.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Try `windows-2016` instead of `windows-latest`

Reply:
That indeed works. Argh, Windows for development sucks.